composer.json

```diff
@@ -1,11 +1,12 @@
 {
-    "name":              "hostnet/accessor-generator-plugin-lib",
+    "name":              "bouw7/accessor-generator-plugin",
```

Comment:
You cannot change the package name. If you wish to do so, please make a fork and adapt this lib to your companies expectations.

Reply:
Sorry Mitchell :joy: Ik lag te slapen.

Reply:
Goedemorgen dan maar @haroldiedema :coffee: